gnovm/tests/file_test.go

```diff
@@ -37,6 +37,16 @@ func TestChallenges(t *testing.T) {
 	runFileTests(t, baseDir, nil)
 }
 
+func TestTypes(t *testing.T) {
+	baseDir := filepath.Join(".", "files/types")
+	runFileTests(t, baseDir, []string{"*_native*"})
+}
+
+func TestTypesNative(t *testing.T) {
+	baseDir := filepath.Join(".", "files/types")
+	runFileTests(t, baseDir, []string{"*_stdlibs*"}, WithNativeLibs())
+}
+
```

Comment:
Ray's PR has been meged, which supports additional directories; please remove these additional tests, the files will be tested with TestStdlibs/TestNative now.